programs/validator-bonds/src/state/settlement_claims.rs

```diff
@@ -0,0 +1,133 @@
+use crate::utils::BitmapProjection;
+use anchor_lang::prelude::Pubkey;
+use anchor_lang::prelude::*;
+pub use anchor_lang::solana_program::entrypoint::MAX_PERMITTED_DATA_INCREASE;
+use anchor_lang::solana_program::system_instruction::MAX_PERMITTED_DATA_LENGTH;
+use std::fmt::Debug;
+
+const SETTLEMENT_CLAIMS_ANCHOR_HEADER_SIZE: usize =
```

Comment:
Why not just import? Also, it should probably be the other way around. The constant defined in here and constants module re-exporting (if needed at all).

In the current setup there should be a big warning comment that the crate::contants::SETTLEMENT_CLAIMS_ANCHOR_HEADER_SIZE must be changed everytime this structure is changed in this file

Reply:
Ok, I see, thanks. That was my wrong consideration for this `const` definition.

I want to leave the constant in `constant.rs` as the place where all constants exported to JSON IDL are defined (`#[constant]`).

I removed the re-export and added the warning.